### ISSUE TITLE
Unit tests fail with raven/raven 0.13.0

### ISSUE THREAD

**Issue by bd808** [Seldaek/monolog#659]
```
$ ./vendor/bin/phpunit tests/Monolog/Handler/RavenHandlerTest.php
PHPUnit 4.6.6 by Sebastian Bergmann and contributors.

Configuration read from /Users/bd808/projects/3rd-party/monolog/phpunit.xml.dist

.....F....

Time: 169 ms, Memory: 5.25Mb

There was 1 failure:

1) Monolog\Handler\RavenHandlerTest::testUserContext
null does not match expected type "array".

/Users/bd808/projects/3rd-party/monolog/tests/Monolog/Handler/RavenHandlerTest.php:120

FAILURES!
Tests: 10, Assertions: 14, Failures: 1.

$ composer show --installed
aws/aws-sdk-php                   2.8.5              AWS SDK for PHP - Use ...
doctrine/annotations              v1.2.4             Docblock Annotations P...
doctrine/cache                    v1.4.1             Caching library offeri...
doctrine/collections              v1.3.0             Collections Abstractio...
doctrine/common                   v2.5.0             Common Library for Doc...
doctrine/couchdb                  dev-master 9f92d8e CouchDB Client
doctrine/inflector                v1.0.1             Common String Manipula...
doctrine/instantiator             1.0.4              A small, lightweight u...
doctrine/lexer                    v1.0.1             Base library for a lex...
graylog2/gelf-php                 1.2.3              A php implementation t...
guzzle/guzzle                     v3.9.3             PHP HTTP client. This ...
jakub-onderka/php-parallel-lint   v0.9               This tool check syntax...
php-console/php-console           3.1.3              Server side PHP librar...
phpdocumentor/reflection-docblock 2.0.4
phpspec/prophecy                  v1.4.1             Highly opinionated moc...
phpunit/php-code-coverage         2.0.16             Library that provides ...
phpunit/php-file-iterator         1.4.0              FilterIterator impleme...
phpunit/php-text-template         1.2.0              Simple template engine.
phpunit/php-timer                 1.0.5              Utility class for timing
phpunit/php-token-stream          1.4.1              Wrapper around PHP's t...
phpunit/phpunit                   4.6.6              The PHP Unit Testing f...
phpunit/phpunit-mock-objects      2.3.0              Mock Object library fo...
psr/log                           1.0.0              Common interface for l...
raven/raven                       0.13.0             A PHP client for Sentr...
ruflin/elastica                   v0.90.10.0         Elasticsearch Client
sebastian/comparator              1.1.1              Provides the functiona...
sebastian/diff                    1.3.0              Diff implementation
sebastian/environment             1.2.2              Provides functionality...
sebastian/exporter                1.2.0              Provides the functiona...
sebastian/global-state            1.0.0              Snapshotting of global...
sebastian/recursion-context       1.0.0              Provides functionality...
sebastian/version                 1.0.5              Library that helps wit...
swiftmailer/swiftmailer           v5.4.0             Swiftmailer, free feat...
symfony/event-dispatcher          v2.6.7             Symfony EventDispatche...
symfony/yaml                      v2.6.7             Symfony Yaml Component
videlalvaro/php-amqplib           v2.5.0             This library is a pure...

$ composer update raven/raven --prefer-lowest
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing raven/raven (0.13.0)
  - Installing raven/raven (0.11.0)
    Loading from cache

Writing lock file
Generating autoload files

$ ./vendor/bin/phpunit tests/Monolog/Handler/RavenHandlerTest.php
PHPUnit 4.6.6 by Sebastian Bergmann and contributors.

Configuration read from /Users/bd808/projects/3rd-party/monolog/phpunit.xml.dist

..........

Time: 93 ms, Memory: 5.00Mb

OK (10 tests, 18 assertions)
```

**Comment by bd808** [comment]
Looks to have been caused by changes introduced in https://github.com/getsentry/raven-php/commit/ab6fb93e34a9de8c53caf9efde4f357c98b1e3d2

**Comment by Seldaek** [comment]
Fixed by 5405308eb1c7dc1fdc0bd408f80eca2ee8ed9a92 - sorry I just forgot to push it :)
